pylint/testutils/lint_module_test.py

```diff
@@ -181,8 +185,10 @@ def error_msg_for_unequal_messages(self, actual_messages, expected_messages):
             msg.extend(" %3d: %s" % msg for msg in sorted(missing))
         if unexpected:
             msg.append("\nUnexpected in testdata:")
-            msg.extend(" %3d: %s" % msg for msg in sorted(unexpected))
+            msg.extend(" %3d: %s" % msg for msg in sorted(unexpected))  # type: ignore
         error_msg = "\n".join(msg)
+        error_msg += "\n\nActual pylint output for this file:\n"
+        error_msg += "\n".join(str(o) for o in actual_output)
```

Comment:
I just noticed this change while working on some tests. IMO it makes it harder rather than easier to quickly see all the errors. Especially when dealing with more than one failing test case at a time.

@Pierre-Sassoulas What do you think?

Reply:
It's possible to disable them individually in functional tests, but it's impossible to test some of them in other way. ``use-symbolic-message-instead`` for example cannot be unit tested because it requires all the other messages to be stored in the message store to work.

Reply:
I might be missing something, but the output messages are already printed if no `unexpected` or `missing` messages are present.

Would this change work?
```py
        if not (missing or unexpected):
            error_msg += "\n\nActual pylint output for this file:\n"
            error_msg += "\n".join(str(o) for o in actual_output)
```

Reply:
Ho sorry, I answered this completely wrong. I thought you were talking about the whole MR.

I added the actual output to functional test to be able to change the test faster when there was a ton of new ``use-symbolic-message-instead`` and the expected messages were wrong in the file.  Because an unexpected ``use-symbolic-message-instead`` does not tell you what is the symbolic message for W0111, you have to see the actual output for that. And you did not see it unless the first part of the functional test succeed. We might have the same problem if we enable ``useless-suppression`` later on.